### DISCUSSION TITLE
1.5 Aurora

### DISCUSSION THREAD

**Discussion by fawstoar** [redeclipse/base#1361] [Help and Support] [answered]
Hi all - is it still possible to download the 1.5 (Aurora) version of the game? Or any other really ancient versions? I'd be interested to try them for old times' sake (and I secretly liked the weapon balance better then :P).

**Comment by Jigoku** [comment]
Hi. You can find old releases of the game here in the "oldsvnre" repo;
https://github.com/redeclipse/oldsvnre/releases

Unfortunately 1.5 doesn't seems to be there, but these previous versions of the game are available to download as release tarballs/zip archives:
* v1.0.0
* v1.2.0
* v1.3.0
* v1.4.0

**Comment by qreeves** [comment, marked as answer]
1.5 would be after the move to GitHub, so it's on the main releases archive: https://github.com/redeclipse/base/releases

**Reply by fawstoar** [reply]
Thanks! Appreciate you keeping all the releases up all this time, quin

I've been messing around in bot matches in 1.2, I guess that's when I was mostly playing this game since the movement and weapons from that version feel really right to me for some reason. Is it just me or did all the weapons use to have much higher lethality?

I do see how 1.2 is less tightly balanced, and having loadouts instead of weapon pickups makes a bit more sense from a competitive balancing standpoint. But gosh I miss the old days when everything felt a little more chaotic and little more overpowered lol. Powerslides used to have crazy length.

Anyway, don't pay me any mind, I'm just feeling nostalgic. Maybe I'll try to set up a 1.2 server for some friends...

(Whoops, accidentally posted this as a separate answer -fixed now! Sorry, I'm new to posting on github).

**Reply by qreeves** [reply]
No problem @fawstoar - nice to see you back around these parts 😄 

Yes, weapons were a bit more lethal back in the old days, and we're working on improving the time-to-kill in the development version. v2.0's weapons are pretty weak, but having ammo that needs to be picked up now helps use balance the weapons even better.
